### ISSUE TITLE
download TODO comment: handle collision when writing solution files

### ISSUE THREAD

**Issue by jdsutherland** [exercism/cli#795]
In effort to resolve TODO comments, I'm in the process of promoting them to issues.

https://github.com/exercism/cli/blob/a86f82974270d2da2090a02437560897a216abc3/cmd/download.go#L198-L199

It seems like https://github.com/exercism/cli/issues/457 is related. Is this still pending further info?

**Comment by kytrinyx** [comment]
I've come to the conclusion that I don't want to make the CLI interactive if we can avoid it (at least not until we've worked out all the existing kinks in a non-interactive way).

I think the solution here would be to display an error message if there's a collision, including a pointer at how you could resolve it.

**Comment by haguro** [comment]
With the premiss of an interactive CLI out of scope for the time being, and the `--force` flag having been added in #979. I believe this should count as resolved?
